xc/xenops_server_xen.ml

```diff
@@ -3129,7 +3180,7 @@ module Actions = struct
         let open Xenctrl in
         let id = Uuidm.to_string (uuid_of_di di) in
         let update = match kind with
-          | "vbd" | "vbd3" ->
+          | "vbd" | "vbd3" | "qdisk3" ->
```

Comment:
Here you use `qdisk3`, but in other files you use `qdisk`. Is that expected? Can you add a comment pointing it out?

Reply:
I'm not sure, this was written by someone else. I couldn't find `qdisk3` elsewhere in the code. I'll try to figure out why this is here.

Reply:
Aha, this branch is not executed:
```
Mar 19 15:58:01 localhost xenopsd-xc: [debug|localhost.localdomain|6 ||xenops] Unknown device kind: 'qdisk'
```
I wonder what the significance of that is. I'll try changing this to `qdisk`.

Reply:
I'll actually try removing this commit and test if it still works.

Reply:
I think we should not do random tests but this should be discussed in your team and you should agree on what this line should do and why

Reply:
We have discussed with Stefano that we'll test it without that commit, and remove it if it's not necessary, but we'll also discuss this tomorrow with Jon.

Reply:
Any update on this?

Reply:
We've looked at it with stefanop, 'qdisk3' is wrong and we were missing some events due to that and lots of log entries complaining about 'Unknown device kind', see my fixup commit.